common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockTransferor.java

```diff
@@ -86,104 +90,114 @@ void createAndStart(String[] blockIds, BlockFetchingListener listener)
   private int retryCount = 0;
 
   /**
-   * Set of all block ids which have not been fetched successfully or with a non-IO Exception.
+   * Set of all block ids which have not been transferred successfully or with a non-IO Exception.
    * A retry involves requesting every outstanding block. Note that since this is a LinkedHashSet,
    * input ordering is preserved, so we always request blocks in the same order the user provided.
    */
   private final LinkedHashSet<String> outstandingBlocksIds;
 
   /**
-   * The BlockFetchingListener that is active with our current BlockFetcher.
+   * The BlockTransferListener that is active with our current BlockFetcher.
    * When we start a retry, we immediately replace this with a new Listener, which causes all any
    * old Listeners to ignore all further responses.
    */
-  private RetryingBlockFetchListener currentListener;
+  private RetryingBlockTransferListener currentListener;
 
   private final ErrorHandler errorHandler;
 
-  public RetryingBlockFetcher(
+  /**
+   * Term indicating whether this RetryingBlockTransferor is for block fetch or push. Useful for
+   * printing more meaningful logs.
+   */
+  private final String transferTerm;
+
+  public RetryingBlockTransferor(
       TransportConf conf,
-      RetryingBlockFetcher.BlockFetchStarter fetchStarter,
+      BlockTransferStarter transferStarter,
       String[] blockIds,
-      BlockFetchingListener listener,
+      BlockTransferListener listener,
       ErrorHandler errorHandler) {
-    this.fetchStarter = fetchStarter;
+    this.transferStarter = transferStarter;
     this.listener = listener;
+    this.transferTerm = (listener instanceof BlockFetchingListener) ?
```

Comment:
Right now this could be either be "fetch" or "push". 
I think we have introduced the `BlockTransferListener` which could now also be extended for some future use-case. 
So I am wondering whether this `transferTerm` should be passed in the constructor instead of assuming that it could be just push or fetch.
Another option would be to add a method to `BlockTransferListener`, let's say, `getLogPrefix` which by default can return `transfer`, but `BlockFetchingListener` can override it to `fetch` and `BlockPushListener` can override it to `push`.

Also, at other places in the comments, we are just assuming push or fetch. Maybe we just say "transfer" in the comments.